src/main/java/com/metriport/fhir/SimplifiedOAuthAuthorizationInterceptor.java

```diff
@@ -14,82 +14,83 @@
 import ca.uhn.fhir.rest.server.interceptor.auth.RuleBuilder;
 
 /**
- * A simplified AuthorizationInterceptor that relies on an upstream JWT validator.
- * This class does not validade the JWT is signed accordingly, it assumes this is
+ * A simplified AuthorizationInterceptor that relies on an upstream JWT
+ * validator.
+ * This class does not validade the JWT is signed accordingly, it assumes this
+ * is
  * verified by an upstream service like API Gateway + Lambdas.
- * It only decodes the JWT and verifies the user has access to the requested resource
+ * It only decodes the JWT and verifies the user has access to the requested
+ * resource
```

Comment:
resulting spacing here is odd with some single-word lines

Reply:
Just applied the formatting coming with the project, not sure why its not formatted like this from the source, maybe they don't enforce it?

But I see your point, updated to be more readable.